### DISCUSSION TITLE
Scale for coloring scheme in 3D panel

### DISCUSSION THREAD

**Discussion by k-man80** [foxglove/community#37] [Q&A]
One of the biggest strength of foxglove is to define color schemes for different data fields.
It would be nice to have a scale of this scheme at the border of the canvas. It should name the used field and it should be possible to define the minor and major scale markings. For each major the corresponding value should be displayed. Also min/max values should be directly shown (especially if you use auto).

**Comment by jtbandes** [comment]
Are you primarily talking about point clouds, or other types of data?

By "a scale of this scheme at the border of the canvas", do you mean a legend?

**Reply by k-man80** [reply]
Yes the name legend might fit better. I personally use mostly point clouds (lidar, radar, SfM 3D points).
